### ISSUE TITLE
[Request]  Snapturning 

### ISSUE THREAD

**Issue by neitey** [Fishbiter/gz3doom#4]
I would suggest to add Snapturning to the locomotion options.

**Comment by neitey** [comment]
This is an amazing mod by the way. Great bringing BD to VR. Most fun I had in month.
Also great fan on fully possesed and quakespasmVR. Any plans of porting some of id's classics to the Quest?

**Comment by RashataanVuul** [comment]
Yes! This mod needs snapturning. The smooth turning makes me sick :(

**Comment by jdawgzim** [comment]
Console scripted snap-turning: 
```
[Doom.ConsoleAliases]  
Name=turn45left  
Command=alias turn45_step "wait 5;-left;turnspeeds 640 1280 320 320;alias turn45_step";turn45_step;wait;turnspeeds 2048 2048 2048 2048;+left
Name=turn45right  
Command=alias turn45_step "wait 5;-right;turnspeeds 640 1280 320 320;alias turn45_step";turn45_step;wait;turnspeeds 2048 2048 2048 2048;+right  
 
[Doom.Bindings]  
rstickright=turn45right  
rstickleft=turn45left  
```
Have to unmap your normal turning in game menus and only use this for turning.

**Comment by wadim-al** [comment]
@jdawgzim Could you please tell exactly what should be unmapped and what else should be done so this script starts working? Can I unmap in config file not to use menus, because they are too sensitive for me and I mess up things quickly in menus?

**Comment by jdawgzim** [comment]
https://www.reddit.com/r/virtualreality/comments/cwry4b/brutal_doom_vr_oneclick_installpack_easy_setup_in/eykfsik?utm_source=share&utm_medium=web2x

Here is the reddit comment where I elaborate more what I do to make the game more comfortable.  Another option instead of snap turn is to just turn up the turning speed and flick the thumbsticks to turn fast and reduce nausea.
